### PR TITLE
16.0 [MIG] l10n_be_partner_kbo_bce

### PR DESCRIPTION
**Pull request by victor-champonnois**
~~Tests do not work because https://github.com/OCA/partner-contact/pull/1397 needs to be merged first~~
[fixed]~~When running `pre-commit run -a`~~
~~I have a prompt that asks me to login to gitlab :~~

### CODE REVIEW

**Comment by luc-demeyer** [comment]
@victor-champonnois 
I think a rebase will fix your problem, cf. https://github.com/OCA/l10n-belgium/commit/aa5e54853173f63db680b1d3191439684a941b97

**Comment by victor-champonnois** [comment]
@luc-demeyer thanks, it works :)

**Comment by luc-demeyer** [comment]
@victor-champonnois 
Thanks.
Still a small issue since this module depends on OCA partner_identification which is not available for 16.0 yet.
Hence you should port that one first or wait until it's available before we can merge your work here.

**Review by remytms — APPROVED** [review]
LGTM

**Comment by marielejeune** [comment]
Hi @victor-champonnois 
The migration of partner_identification is ready and just waits for 1 more review before being merged.
Please see https://github.com/OCA/partner-contact/pull/1397
Thanks!

**Comment by sbidoul** [comment]
@victor-champonnois can you rebase, now that partner_identification is merged?

File: l10n_be_partner_kbo_bce/tests/test_l10n_be_partner_kbo_bce.py
Comment (carmenbianca) [inline l10n_be_partner_kbo_bce/tests/test_l10n_be_partner_kbo_bce.py]: Should use setUpClass, which is more performant.

File: l10n_be_partner_kbo_bce/tests/test_l10n_be_partner_kbo_bce.py
Comment (carmenbianca) [inline l10n_be_partner_kbo_bce/tests/test_l10n_be_partner_kbo_bce.py]: can use plain `super()`

**Review by carmenbianca — CHANGES_REQUESTED** [review]
Small request for change.

File: l10n_be_partner_kbo_bce/tests/test_l10n_be_partner_kbo_bce.py
Comment (victor-champonnois) [reply l10n_be_partner_kbo_bce/tests/test_l10n_be_partner_kbo_bce.py]: I tested with setupclass instead of super(), but the tests failed. They are not designed to be used with setupclass, it would demand to rewrite them. And I think this is out of scope

File: l10n_be_partner_kbo_bce/tests/test_l10n_be_partner_kbo_bce.py
Comment (victor-champonnois) [reply l10n_be_partner_kbo_bce/tests/test_l10n_be_partner_kbo_bce.py]: done

**Review by carmenbianca — APPROVED** [review]

**Comment by sbidoul** [comment]
Thanks!

**Pull request merged**
